src/input-select/input-select.tsx

```diff
@@ -157,7 +159,9 @@ export const InputSelect = <T, V>({
                 </PlaceholderLabel>
             );
         } else {
-            return (
+            return renderCustomSelectedItem ? (
+                renderCustomSelectedItem(getDisplayValue())
```

Comment:
I think passing in the original option would provide the most flexibility, if other properties are needed for rendering